mesonbuild/backend/ninjabackend.py

```diff
@@ -27,6 +27,7 @@
 from ..arglist import CompilerArgs
 from ..compilers import Compiler
 from ..linkers import ArLikeLinker, RSPFileSyntax
+from ..linkers.linkers import VisualStudioLikeLinker
```

Comment:
It would be preferable to only import this inside of `generate_link()` as then we know we are actually linking native code and have surely found a concrete linker impl, so consequently this module will have already been imported before.

Importing it unconditionally during backend generation means that it slows down all project generation no matter what, and we've put quite a bit of effort into avoiding loading too much of meson's own internal code unless it's actually used.